geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/StartMemberUtilsTest.java

```diff
@@ -101,6 +105,33 @@ public void testGeodeOnClasspathIsFirst() {
     assertThat(gemfireClasspath).startsWith(customGeodeCore);
   }
 
+  @Test
+  public void testAddMaxHeap() {
+    List<String> baseCommandLine = new ArrayList<>();
+
+    // Empty Max Heap Option
+    StartMemberUtils.addMaxHeap(baseCommandLine, null);
+    assertThat(baseCommandLine.size()).isEqualTo(0);
+
+    StartMemberUtils.addMaxHeap(baseCommandLine, "");
+    assertThat(baseCommandLine.size()).isEqualTo(0);
+
+    // Only Max Heap Option Set
+    StartMemberUtils.addMaxHeap(baseCommandLine, "32g");
+    assertThat(baseCommandLine.size()).isEqualTo(3);
+    assertThat(baseCommandLine).containsExactly("-Xmx32g", "-XX:+UseConcMarkSweepGC",
+        "-XX:CMSInitiatingOccupancyFraction=" + StartMemberUtils.CMS_INITIAL_OCCUPANCY_FRACTION);
+
+    // All Options Set
+    List<String> customCommandLine = new ArrayList<>(
+        Arrays.asList("-XX:+UseConcMarkSweepGC", "-XX:CMSInitiatingOccupancyFraction=30"));
+    StartMemberUtils.addMaxHeap(customCommandLine, "16g");
+    System.out.println(customCommandLine);
```

Comment:
just get rid of the println call. Everything else looks good.

Reply:
My bad... thanks @jinmeiliao, I'll remove the `System.out.println` call and push the change. Cheers.